locale/lang-tools/merge.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python3
+
+import argparse
+import toml
+import sys
+import os
+
+## Unmerges a file
+def unmerge(path):
+    p = path.split('/')
+    print(p)
+    print(p[-2])
+    if '-' not in p[-2]:
+        return toml.load(path)
+    p[-2] = p[-2].split('-')[0]
+    new_path = '/'.join(p)
+    if not os.path.exists(new_path):
+        return toml.load(path)
```

Comment:
shouldn't we throw error here?

Reply:
No, if a file has no parent the unmerge just loads it

Reply:
aren't files with no parents (so undashed files) returning earlier in the function?

Reply:
There could be two languages not merged